js/test/domain/userPreferencesFactory.spec.ts

```diff
@@ -363,13 +363,13 @@ describe('userPreferencesFactory', () => {
       expect(new Date().toISOString()).toEqual(123);
     });
     it('cookies should be settable and gettable', () => {
-      expect(Cookies.getJSON('abcdef')).toBeUndefined();
+      expect(Cookies.get('abcdef')).toBeUndefined();
       Cookies.set('abcdef', 'this is my cookie value');
-      expect(Cookies.getJSON('abcdef')).toBe('this is my cookie value');
+      expect(Cookies.get('abcdef')).toBe('this is my cookie value');
     });
     it('cookies should be settable with an object', () => {
-      Cookies.set('abcdef', { thisIs: 'AnObject' });
-      expect(Cookies.getJSON('abcdef')).toEqual({ thisIs: 'AnObject' });
+      Cookies.set('abcdef', JSON.stringify({ thisIs: 'AnObject' }));
+      expect(JSON.parse(Cookies.get('abcdef'))).toEqual({ thisIs: 'AnObject' });
     });
```

Comment:
not really sure why we have these tests historically

https://github.com/hmrc/tracking-consent-frontend/blame/7f52666b621f37304bc8545e759d29f03f105da9/js/test/domain/userPreferencesFactory.spec.ts#L365-L374

before each test we mock Cookies.set / Cookies.get[JSON], so we are testing our mocks work?

Reply:
looking at the date one, that seems to be what we're doing, these are just making sure the fake stuff behaves how we expect, so the JSON.stringify / parse stuff here isn't needed